### PR TITLE
[Sources] Fix add source button crash on iOS 13 when no URL is in cli…

### LINKED ISSUE #1966: Crash when trying to add repo
**Describe the bug**
Zebra crashes when I press the button to add a repo

**To Reproduce**
Open Zebra
Go to sources tab
Press + button
Crash

**Expected behavior**
No crash

**Screenshots**
https://twitter.com/opa334dev/status/1494806842710581248

**Device Information:**
 - iOS Version: 13.7
 - Zebra Version: 1.1.25
 - Device Type: iPhone X
 - Jailbreak Type: checkra1n

**Crash Log or System Log**
https://pastebin.com/dLVfgPzU

**Additional context**
This worked fine earlier, now it no longer does

### PR DESCRIPTION
**Pull request by opa334**
…pboard (Fixes #1966)

### CODE REVIEW

**Comment by kirb** [comment]
Thanks! I did fix this locally but didn’t push for some reason. I’ll just merge your commit instead although I’ll need to rebase to match up the codestyle, brace should be on the same line. (Feel free to beat me to it and fix that yourself if you’d like)

**Review by PoomSmart — DISMISSED** [review]

**Comment by opa334** [comment]
done!

File: Zebra/Tabs/Sources/Controllers/ZBSourceListTableViewController.m [hunk 1]
```diff
@@ -374,9 +374,12 @@ - (void)_detectURLOnPasteboardWithCompletion:(void (^)(NSString *))completion {
     } else {
         NSDataDetector *detector = [NSDataDetector dataDetectorWithTypes:NSTextCheckingTypeLink error:nil];
         NSString *input = [UIPasteboard generalPasteboard].string ?: @"";
-        NSString *result = [input substringWithRange:[detector rangeOfFirstMatchInString:input options:kNilOptions range:NSMakeRange(0, input.length)]];
-        if ([self _isValidURL:result checkFullyQualified:NO]) {
-            completion([self _fullyQualifiedURLForInput:result]);
+        NSRange range = [detector rangeOfFirstMatchInString:input options:kNilOptions range:NSMakeRange(0, input.length)];
+        if(range.location != NSNotFound) {
```
Comment (PoomSmart) [inline Zebra/Tabs/Sources/Controllers/ZBSourceListTableViewController.m:378]:
```suggestion
        if (range.location != NSNotFound) {
```
:D

**Review by PoomSmart — APPROVED** [review]

**Review by kirb — APPROVED** [review]
Thanks!

**Pull request merged**